tests/unit/test_clidriver.py

```diff
@@ -1077,5 +1093,20 @@ def test_handle_exception_in_main(self):
         self.assertIn('error', fake_stderr.getvalue())
 
 
+class TextCreateCLIDriver(unittest.TestCase):
+    def test_create_cli_driver_parse_args(self):
+        stderr = io.StringIO()
+        with contextlib.redirect_stderr(stderr):
+            driver = create_clidriver(['--profile', 'foo', '--debug'])
+        profile = driver.session.get_config_variable('profile')
+        self.assertEqual(profile, 'foo')
+        self.assertIn('CLI version', stderr.getvalue())
+
+    def test_create_cli_driver_wo_args(self):
+        driver = create_clidriver([])
+        profile = driver.session.get_config_variable('profile')
+        self.assertIsNone(profile)
```

Comment:
I'd be hesitant to asserting `None` here without patching out the environment. Specifically, the profile can also be set through environment variables and would probably cause this test to fail if they got set.